setup.py

```diff
@@ -11,7 +11,7 @@
     long_description = fd.read()
 
 setuptools.setup(
-    name='irisapi',
+    name='iris',
```

Comment:
the name irisapi is tied to the pypi project, do not change it here: https://pypi.org/project/irisapi/

Reply:
changes done